Backend/dev/test-tool/dashboard/src/components/CollectionRunner.css

```
@@ -232,3 +232,104 @@
+.jv-close-btn:hover { color: #f85149; }
+
+/* JSON tree */
+.jv-tree {
+  overflow: auto;
+  padding: 12px 16px;
+  font-family: 'SF Mono', 'Cascadia Code', 'Fira Code', monospace;
+  font-size: 12px;
+  line-height: 1.6;
+}
+.jv-indent { padding-left: 20px; }
+.jv-entry { }
```

Stylelint (17.7.0)

[error] Empty block (block-no-empty)

(block-no-empty)